src/storage/storage.cpp

```diff
@@ -12,29 +12,194 @@
  * Public
  **********************************************************************************************************************/
 
+constexpr const char* kStorageInstancePath = CONFIG_AOS_STORAGE_PATH "/database_instance";
```

Comment:
Move it privat section of class. Why service.db but instance without ext?
Use prefix c for const.